### DISCUSSION TITLE
Fatal error | flag error: report flag error: unable to parse flag: invalid argument

### DISCUSSION THREAD

**Discussion by princechennai** [aquasecurity/trivy#7743] [Q&A] [answered]
### Question

I'm using [this document](https://aquasecurity.github.io/trivy/v0.56/tutorials/integrations/gitlab-ci/#gitlab-ci-using-trivy-container) to setup container image scan on gitlab ci.
While running the image scan using this command
$TRIVY_REPORT=container-report.json
./trivy image --exit-code 0 --format json --template "@/contrib/gitlab-codequality.tpl" -o $TRIVY_REPORT "ubuntu:latest"
I get the below error
FATAL	Fatal error	flag error: report flag error: unable to parse flag: invalid argument "container-report.json" for "--report" flag: must be one of ["all" "summary"]

### Target

Container Image

### Scanner

Vulnerability

### Output Format

JSON

### Mode

Standalone

### Operating System

Ubuntu

### Version

```bash
0.56.2
```

**Comment by knqyf263** [comment, marked as answer]
`TRIVY_XXX` is treated as trivy options.
https://aquasecurity.github.io/trivy/v0.56/docs/configuration/#environment-variables

Renaming `TRIVY_REPORT` to `REPORT_FILE` or something like that should fix the error.

**Reply by princechennai** [reply]
@knqyf263 Yup. that worked. But the content of the REPORT_FILE has one line saying `/contrib/gitlab.tpl`

**Reply by knqyf263** [reply]
Try `--format template`
https://aquasecurity.github.io/trivy/v0.56/docs/configuration/reporting/#template

**Reply by princechennai** [reply]
that is what I'm trying
`trivy image --exit-code 0 --vuln-type os --scanners vuln --ignore-unfixed --format template --template "/contrib/gitlab.tpl" --output "$CI_PROJECT_DIR/$REPORT_FILE" "$IMG_NAME"`

**Reply by knqyf263** [reply]
`@` is missing. Please reference the doc.

**Reply by princechennai** [reply]
Okay.. Sorry, I did not tell you the full story
I'm using this command to get the file locally `wget https://raw.githubusercontent.com/aquasecurity/trivy/refs/heads/main/contrib/gitlab.tpl -O contrib/gitlab.tpl` otherwise I get this error `Fatal error	report error: unable to write results: failed to initialize template writer: error retrieving template from path: open /contrib/gitlab.tpl: no such file or directory`
I'm following this documentation
https://aquasecurity.github.io/trivy/v0.56/tutorials/integrations/gitlab-ci/

**Reply by princechennai** [reply]
Alright, actually, `"@contrib/gitlab.tpl"` worked. Need to update the document
